docs/04-resources/04-tutorials/governance-dashboard.md

```diff
@@ -0,0 +1,370 @@
+
+:::
+
+* **How to get it**:  
+  * Deploy the TeamsManager.sol contract to the Rootstock network using RemixIDE, a popular tool for writing and deploying smart contracts.   
+  * Once deployed, you’ll find the contract’s address in the Remix’s console. Copy this address.  
+  * Paste this address into your .env file as shown below:
+
+```
+NEXT_PUBLIC_TEAM_MANAGER_ADDRESS=<Your_Team_Manager_Contract_Address>
+```
+
+#### **2. NEXT_PUBLIC_RPC_URL**
+
+This is the RPC (Remote Procedure Call) URL that allows an application to interact with the Rootstock blockchain. An RPC URL is essentially a link to the blockchain network.
+
+:::tip[Tip]
+
+If you’re unsure how to navigate the Rootstock RPC Dashboard, refer to the [Getting Started with Rootstock RPC API](https://dev.rootstock.io/developers/rpc-api/rootstock/setup/).
+
+:::
+
+* **How to get it**:
+
+    * Go to the [Rootstock RPC Dashboard](https://dashboard.rpc.rootstock.io/login) and create an account or log in.  
+    * Once logged in, create a new project or access an existing one to generate a testnet RPC URL.  
+    * Copy the URL and add it to your .env file like so:
+
+```
+NEXT_PUBLIC_RPC_URL=<Your_Rootstock_RPC_URL>
+```
+
+#### **3. NEXT_PUBLIC_EXPLORER**
+
+This is the blockchain explorer URL, which lets you view transaction details on the Rootstock network.
+
+* **How to get it**:  
+  * For testnet (for testing purposes), use this URL: https://explorer.testnet.rootstock.io.  
+  * For mainnet (for production), use this URL: https://explorer.rootstock.io.
+
+**Example**:
+
+```
+NEXT_PUBLIC_EXPLORER=https://explorer.testnet.rootstock.io
+```
+
+ This variable allows your app to link to transaction history directly on the explorer, providing a user-friendly way to verify actions on the blockchain.
+
+#### **4. NEXT_PUBLIC_PINATA_URL**
+
+This is the URL to Pinata, an IPFS (InterPlanetary File System) provider, which will be used to store and retrieve metadata related to teams and votes.
+
+<figure>
+<img src="/img/resources/tutorials/governance-dashaboard/NEXT_PUBLIC_PINATA_URL.png" alt="NEXT_PUBLIC_PINATA_URL"/>
+  <figcaption>NEXT_PUBLIC_PINATA_URL (fig 2.)</figcaption>
+</figure>
+
+* **How to get it**:  
+  * Start by logging into your Pinata Cloud account and navigate to the dashboard [Pinata](https://www.pinata.cloud/).  
+  * In the dashboard, click on the **"Files"** tab to access the file storage area.  
+  * Click the **"Upload"** button and select a sample image file from your device to upload.  
+  * After the upload is complete, locate the uploaded image in the file list. You’ll see a **URL** provided by Pinata that can be used to access your file.  
+
+For example:
+
+```
+https://emerald-familiar-cobra-431.mypinata.cloud/ipfs/QmVsBy3vo9tmbjYmFNCiRwAvCBU7p9peEEgyR9awJg4Tyj
+```
+
+  * The base URL is the part of the file URL before the unique CID (Content Identifier). In this example, the base URL is:
+
+```
```

Comment:
Remove url from code block, use a link instead. Same with above and below example